cg/apps/shipping/shipping_api.py

```diff
@@ -52,3 +52,12 @@ def deploy_shipping(self):
         """
         LOG.info("Deploying the shipping software")
         self.deploy(app_name="shipping")
+
+    def deploy_genotype(self):
+        """Deploy the tool genotype
+
+        Deployment of genotype does not need any extra information since it is following the regular workflow
+        with conda environments using standard names and pip
+        """
+        LOG.info("Deploying the genotype software")
```

Comment:
this could be moved to deploy method - f"deploying the {app_name}", will probably matter a bit more once we deploy every package this way

Reply:
Yes I think you are right 👍

Reply:
Fixed according to your suggestion